doc/classes/AnimationPlayer.xml

```diff
@@ -217,7 +219,8 @@
 			<argument index="0" name="reset" type="bool" default="true">
 			</argument>
 			<description>
-				Stop the currently playing animation. If [code]reset[/code] is [code]true[/code], the anim position is reset to [code]0[/code].
+				Stop the currently playing animation. If [code]reset[/code] is [code]true[/code], the animation position is reset to [code]0[/code].
```

Comment:
Since it needs an amend for the typo below, maybe you can add this to this sentence?
"If [code]reset[/code] is [code]true[/code], the animation position is reset to [code]0[/code] **and the playback speed is reset to 1.0**."
(without emphasing in the docs, it's just so that you see the edit)

Reply:
Done.